backend/src/main/java/com/shipping/backend/services/ShippingRequestSenderImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.shipping.backend.services;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
Is this a `@Service` or a `@Component`?

Reply:
I change it to a component as there is no bussines logic on it